docs/manifest.json

```diff
@@ -3447,6 +3447,10 @@
                         {
                           "title": "`<SignOutButton>`",
                           "href": "/docs/reference/components/unstyled/sign-out-button"
+                        },
+                        {
+                          "title": "`<UNSAFE_PortalProvider>`",
+                          "href": "/docs/reference/components/unstyled/portal-provider"
```

Comment:
Wondering if we should have a new section `Utilities` vs this living within `Unstyled`